servers/rendering/renderer_rd/storage_rd/light_storage.cpp

```diff
@@ -1732,11 +1732,12 @@ void LightStorage::update_reflection_probe_buffer(RenderDataRD *p_render_data, c
 		if (!rpi) {
 			continue;
 		}
-
-		Transform3D transform = rpi->transform;
+		ReflectionProbe *probe = reflection_probe_owner.get_or_null(rpi->probe);
+		Vector3 extents = probe->size / 2;
+		float probe_size = Math::sqrt(extents.x * extents.x + extents.y * extents.y + extents.z * extents.z);
```

Comment:
```suggestion
		float probe_size = extents.length();
```
Or am I missing something

Reply:
Oh yeah, you're totally right, that makes a lot more sense 😂